### ISSUE TITLE
Standard connector compatibility

### ISSUE THREAD

**Issue by stevexyz** [thomasheckmann/zx81-external-16k#1]
Ciao Thomas,
I was looking at this nice interface, but for people to use it  would require another component to be attached to the standard ZX81.
I was thinking that if you make a design with holes for the standard connector you would have the compatibility with both options (current and standard connector). It is just an idea, feel free to close the issue and discard it in case.
To give you the idea I made the mockup picture attached.
![mockup](https://github.com/user-attachments/assets/1b5d8cb6-c2ab-44b7-9af1-b32c4ced4833)
PS: and please no SMD :)
Best!
_Stefano

**Comment by thomasheckmann** [comment]
Thanks for your feedback, the original idea is to have either the standard connector or the pads to using my bus expander - as I understand it, you suggest having both connection options as the same time?

**Comment by stevexyz** [comment]
Yes. As depicted probably is feasible with a single design.  Not sure if the bars of the other connector should be shortened just a bit in order to allow clearance for the bottom of the case
